backend/svirt.pm

```diff
@@ -179,7 +178,8 @@ sub do_extract_assets ($self, $args) {
     my $first_hdd = $bmwqemu::vars{S390_ZKVM} ? 'a' : 'b';
     my $name = $args->{name};
     my $hdd_num = $args->{hdd_num} - 1;
-    my $svirt_img_name = IMAGE_STORAGE . $self->{vmname} . chr(ord($first_hdd) + $hdd_num) . '.img';
+    my $vmname = $self->console('svirt')->name;
```

Comment:
I would expect that this brings us back to `unable to extract assets: Can't call method "name" on an undefined value`?